.travis.yml

```diff
@@ -164,6 +169,17 @@ jobs:
          - MATRIX_EVAL="CC=clang-8 && CXX=clang++-8"
       script:
         - ./.github/travis/build.sh
+    - stage: Coverity Scan
+      name: "Coverity Scan"
+      env:
+        - _COVERITY_EMAIL="jeanphilippe.legaul@gmail.com"
+        - _COVERITY_URL="https://scan.coverity.com/download/linux64"
+        - _COVERITY_MD5="d0d7d7df9d6609e578f85096a755fb8f"
+        global:
+          # COVERITY_SCAN_TOKEN
+          - secure: "XVDYDFLVmCe+E1lr/SYgzpxUXIIRJyZoBj7tlisGYM922wNlKQP70M5CPxndR0AegVCe8HAN0Rh1O3fiELwSPg+iTnXuqwsGjLlr8oQhJ27reXVxJqwWHw7WylTmbtqiOvO8MTcurCgEOYzXMUW1h4DAuI7W2+sTgovaYOicH0kEK3YhhKRMsqil34p8IvfcaWbdf1Wz+eLwUNxZAFld99EfxBVcAxOvX+BwwbrCVGlkspoLpmEnG+nn0f/OwpHMUS6OcImUfTcwDauVtJywXN6xeVPmqwTRft0Vc6Q06myOB8i3+XDp7yVsW04L07rVqnHMEM8Vp53Sd0Eq3QrnjncoyzUVY6izdzg6+6B2cD+7uMZpn2+0y5d5RJfTZNY13AagBrEYjHqrRsAY4cDBpoD0pSpZAtkv/87fU8IPG7VSjr3N3vl26rv3WSVj4eP01askmOHzlDxEaLyHZ6GgjlZnhY1bGqWi03WWCrl6PBK5En1zRqtQggYv2JUuJMT/gN+tD/d5yCgP+H0sqwHi5+JNX57vZqjzQW6yd3+daL+qcUMGgOqRsS3P54AovP70ztF/K4S0j1k4qwwpP8LHlBFDJICHz95VBvyCpSxRBNijigmyJxqY2jwTyMYQnPaAeT3kvU3M9KXtRjaby9y7zGtIZanChNE2/siX5C++1C0="
+      script:
```

Comment:
When the email is changed, the token needs to be regenerated (https://docs.travis-ci.com/user/environment-variables/)
alternatively, it could be set in the Travis UI.

Reply:
You should definitely set via the Travis UI rather than use the .travis.yml file.

Reply:
done :)
thanks